sdk/CHANGELOG.md

```diff
@@ -36,6 +36,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Multiple prepare methods returning `PreparedMintTokenTransaction`: `prepare_mint_native_token` and `prepare_increase_native_token_supply`;
 - Stronghold snapshot migration from v2 to v3;
 - `SecretManage::sign_evm`;
+- `Password` type which derives `Zeroize` and `ZeroizeOnDrop`;
 
 ### Changed
```

Comment:
I would mention that some function now takes an `impl Into<Password>` since it's breaking change

Reply:
`impl Into<Password>` has been removed for the possible security issues you pointed out.